ui/cypress/integration/admin.test.ts

```diff
@@ -238,15 +238,6 @@ describe('Use Admin tab', () => {
       })
 
       it('create a role, edit it, assign it to a user, and delete it', () => {
-        cy.getByTestID('admin-table--head').within(() => {
-          cy.get('th').contains('Users').should('exist')
-        })
-
-        cy.getByTestID('hide-users--toggle').click()
-        cy.getByTestID('admin-table--head').within(() => {
-          cy.get('th').contains('Users').should('not.exist')
-        })
-
```

Comment:
FYI: the deleted code is reworked in https://github.com/influxdata/chronograf/pull/5944, `hide-users--toggle` is assigned misconceptually